Mathlib/LinearAlgebra/AffineSpace/AffineMap.lean

```diff
@@ -513,17 +499,17 @@ theorem linear_bijective_iff (f : P1 →ᵃ[k] P2) :
 theorem image_vsub_image {s t : Set P1} (f : P1 →ᵃ[k] P2) :
     f '' s -ᵥ f '' t = f.linear '' (s -ᵥ t) := by
   ext v
-  rw [Set.mem_vsub] -- Porting note: `simp only` did not use this lemma
-  simp only [Set.mem_image, exists_exists_and_eq_and, exists_and_left, ←
-    f.linearMap_vsub]
+  -- porting note: `simp` needs a placeholder for the `β` argument to `Set.mem_vsub`
+  simp only [Set.mem_vsub (β := _), Set.mem_image,
```

Comment:
This is deeply weird

Reply:
[Zulip thread](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/simp.20doesn't.20work.20without.20placeholders/near/342032263)